tools/quic_trace_to_time_sequence_gnuplot.cc

```diff
@@ -130,10 +137,11 @@ void PrintTimeSequence(std::istream* trace_source) {
 }  // namespace quic_trace
 
 int main(int argc, char* argv[]) {
-  gflags::ParseCommandLineFlags(&argc, &argv, true);
+  absl::ParseCommandLine(argc, argv);
 
-  if (FLAGS_sequence != "send" && FLAGS_sequence != "ack" &&
-      FLAGS_sequence != "loss") {
+  if (absl::GetFlag(FLAGS_sequence) != "send" &&
+      absl::GetFlag(FLAGS_sequence) != "ack" &&
+      absl::GetFlag(FLAGS_sequence) != "loss") {
```

Comment:
In the next line, should it say `--sequence` instead of `--type`?

Reply:
Fixed.